block/modes.go

```diff
@@ -50,10 +50,8 @@ func (m *Manager) runAsProposer(ctx context.Context, eg *errgroup.Group) error {
 	m.RunMode = RunModeProposer
 	// Subscribe to batch events, to update last submitted height in case batch confirmation was lost. This could happen if the sequencer crash/restarted just after submitting a batch to the settlement and by the time we query the last batch, this batch wasn't accepted yet.
 	go uevent.MustSubscribe(ctx, m.Pubsub, "updateSubmittedHeightLoop", settlement.EventQueryNewSettlementBatchAccepted, m.UpdateLastSubmittedHeight, m.logger)
-
-	// Sequencer must wait till the DA light client is synced. Otherwise it will fail when submitting blocks.
-	// Full-nodes does not need to wait, but if it tries to fetch blocks from DA heights previous to the DA light client height it will fail, and it will retry till it reaches the height.
-	m.DAClient.WaitForSyncing()
+	// Subscribe to P2P received blocks events (used for P2P syncing).
```

Comment:
why `m.DAClient.WaitForSyncing()` removed? Having synced DA light client is a precondition for proposer no?

Reply:
this shouldnt be removed. i readded it. although i think there is room for improvement for proposers when da is down (but out of the scope of this pr). i opened an issue for it https://github.com/dymensionxyz/dymint/issues/1254

Reply:
still can't see it

Reply:
forgot to push. should be there now